content/documents/search-index-tools.mdx

```diff
@@ -0,0 +1,73 @@
+- Only one "active" search index per type can exist at any given time, and only an index with a status of `Ready` can be the active index for a marketplace. OrderCloud will point to a new index only after a successful rebuild.
+- Index errors typically arise when xp data types are not consistent between items within the same resource type. See the section on [xp schema errors](/knowledge-base/search-index-tools#understanding-xp-schema-errors) below for more detail.
+- Errors can exist on an active index. OrderCloud will fail an index build if the number of rejected items is considered too high or xp limits are exceeded.
+- A full index rebuild is a tool to "reset" your search index from the master OrderCloud record, and can oftentimes resolve data synchronization issues. This capability was previously limited to only the support team, but is now extended to marketplace administrators [with limitations](/knowledge-base/search-index-tools#rebuilding-your-index).
+
+## Troubleshooting With Search Index Tools
+
+Registered portal users now have visibility to search index details for their assigned marketplaces. After selecting a marketplace from the list, you will have the option to view either Product or Order search indexes from the tab selections. 
+
+<figure class="blog-figure">
+<img src="/images/knowledge-base/inline/search-index-tools/product_index_view.png" alt="Search Index Tools Products View"/>
+</figure>
+
+The details view for a selected search index will display the following:
+- Status - `Ready`, `Building`, `Rebuilding`, `WaitingToBuild`, `Failed`, `Canceled`, or `Invalid`
+- The `Active` badge - indicates the "current" index OrderCloud is pointed to for the selected resource
+- Schema errors (if any) associated with the index
+  - Visibility to error logs allows users to target which items are causing issues for a given index. 
+  - Logs may include the rejected Product, Order, or Line Item ID, the flagged xp field, and its corresponding value.
+  - Error logs are limited to 100 documents for brevity, although there may be more items in the marketplace that have failed to index in Elasticsearch.
+
+>Not seeing a search index for your resource type? You will need to create items for that resource first (reminder that only submitted Orders are indexed) and make a LIST endpoint call. Also keep in mind that premium search for Orders [may not be enabled](/knowledge-base/order-search#how-do-i-test-this) for your Production marketplace if it was created prior to April 20, 2022.
+
+## Understanding Xp Schema Errors
+
+Let's walk through an example of an xp schema inconsistency. Two products are created, both with the same xp field name, but with different datatypes. This inconsistency produces the following error: _"Failed to parse field [xp.FreeShipping] of type [boolean] in document with id 'product2'. Preview of field's value: '0'"_
```

Comment:
we always want to talk about xp as key/value pairs, so be sure to use that language throughout 

xp key name, but with different value types (e.g. an integer and a boolean).